apps/playground-web/src/components/pay/direct-payment.tsx

```diff
@@ -16,9 +16,9 @@ export function BuyMerchPreview() {
         payOptions={{
           mode: "direct_payment",
           paymentInfo: {
-            amount: "15",
-            chain: base,
-            token: getDefaultToken(base, "USDC"),
+            amount: "0.1",
+            chain: sepolia,
+            token: getDefaultToken(sepolia, "USDC"),
             sellerAddress: "0xEb0effdFB4dC5b3d5d3aC6ce29F3ED213E95d675",
```

Comment:
was this meant to stay changed?

Reply:
Yes. Easier to test for ppl. Don't really wanna charge real money